docs/project/contributors.md

```diff
@@ -30,26 +30,17 @@ Blocknet is an open-source project with a team consisting of contributors from a
 	<div class="name">Dan Metcalf&nbsp;&nbsp;
 		<a href="https://www.linkedin.com/in/dan-metcalf-b821a812/" target="_blank"><i class="fa fa-linkedin-square" aria-hidden="true"></i></a>
 	</div>
-	<div class="title">Development</div>
+	<div class="title">Co-Founder and Development</div>
 	<div class="platform">Discord <span class="handle">@atcsecure#6378</span></div>
 </div>
 
 <div class="contributor">
-	<div class="img-round"><img src="/img/contributors/michael-madgett.jpg"></div>
-	<div class="name">Michael Madgett&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/mmadgett/" target="_blank"><i class="fa fa-linkedin-square" aria-hidden="true"></i></a>
-	</div>
-	<div class="title">Development</div>
-	<div class="platform">Discord <span class="handle">@michael#6510</span></div>
-</div>
-
-<div class="contributor">
-	<div class="fa fa-user-circle" aria-hidden="true"></div>
-	<div class="name">Kamel Bent Ahmed&nbsp;&nbsp;
-		<a href="https://www.linkedin.com/in/kamel-bent-ahmed-66211b13/" target="_blank"><i class="fa fa-linkedin-square" aria-hidden="true"></i></a>
+	<div class="img-round"><img src="/img/contributors/arlyn-culwick.jpg"></div>
+	<div class="name">Arlyn Culwick&nbsp;&nbsp;
+		<a href="https://www.linkedin.com/in/arlynculwick/" target="_blank"><i class="fa fa-linkedin-square fa-1x" aria-hidden="true"></i></a>
```

Comment:
Replace with:
<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>

Reply:
Hmm... the code wasn't included above.
Trying again:
<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>

Reply:
`<a href="https://twitter.com/arlynculwick" target="_blank"><i class="fa fa-twitter-square fa-1x" aria-hidden="true"></i></a>`